spec/far_mar/market_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
+
+describe FarMar do
+  describe FarMar::Market do
+    describe "initialize" do
+      it "creates an instance of a Market" do
+        expect(FarMar::Market.new()).to be_an_instance_of FarMar::Market
```

Comment:
Follow what you did with the product and sale and do not have the `new()` with the empty parenthesis